doc/api_assets/style.css

```diff
@@ -1,5 +1,6 @@
 /*--------------------- Layout and Typography ----------------------------*/
 html {
+  -webkit-overflow-scrolling: touch;
```

Comment:
This enables momentum scrolling. The scrolling experience with it on is a lot better but its use is is [discouraged](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling).

I've had no issues while testing but we can remove it if we think it will cause problems.